### PR TITLE
Add some LBRY repos to projects list

### PR DESCRIPTION
**Pull request by ICTman1076**
Not my project, but a good project with a few issues for beginners. Let me know what you think.

### CODE REVIEW

File: _data/projects/lbry-desktop.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: LBRY desktop
+desc: A browser and wallet for LBRY, the decentralized, user-controlled content marketplace.
+site: https://github.com/lbryio/lbry-desktop
+tags:
+- electron
+- node.js
+- javascript
+upforgrabs:
+  name: good first issue
+  link: https://github.com/lbryio/lbry-desktop/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22
```
Comment (shiftkey) [inline _data/projects/lbry-desktop.yml]:
```suggestion
  link: https://github.com/lbryio/lbry-desktop/labels/good%20first%20issue
```

File: _data/projects/lbry-sdk.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: LBRY SDK
+desc: The LBRY SDK for building decentralized, censorship resistant, monetized, digital content apps.
+site: https://github.com/lbryio/lbry-sdk
+tags:
+- python
+- decentralized
+- blockchain
+upforgrabs:
+  name: good first issue
+  link: https://github.com/lbryio/lbry-sdk/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22
```
Comment (shiftkey) [inline _data/projects/lbry-sdk.yml]:
```suggestion
  link: https://github.com/lbryio/lbry-sdk/labels/good%20first%20issue
```

**Review by shiftkey — COMMENTED** [review]
Thanks for submitting this project @ICTman1076 - I added a couple of suggestions to tidy up the links which will help our automation to generate stats. Other than that, everything looks good to me!

**Comment by ICTman1076** [comment]
Just committed your suggestions. Thanks for reviewing :)

**Review by shiftkey — APPROVED** [review]

**Comment by ICTman1076** [comment]
Something you may want to know, LBRY will often [give contributors LBC](https://github.com/ICTman1076/lbry-desktop/blob/master/CONTRIBUTING.md), a cryptocurrency, for helping out.

**Pull request merged**